pkgs/racket-doc/syntax/scribblings/parse/ex-kw-args.scrbl

```diff
@@ -99,6 +99,27 @@ If the second subtemplate of a @racket[~?] template is
 @racket[(~@)]---that is, it produces no terms at all---the second
 subtemplate can be omitted.
 
+@section{Optional Keyword Argument as flags}
```

Comment:
Make sure plurals are consistent. Either plural as plural `Optional Keyword Arguments as flags` or singular as singular `Using an Optional Keyword Argument as a flag`, but not singular as plural.

Reply:
Good catch!